### PR TITLE
Docusaurus

### PR DESCRIPTION
**Pull request by amirhouieh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
+$ USE_SSH=true yarn deploy
+```
+
+Not using SSH:
+
+```
+$ GIT_USER=<Your GitHub username> yarn deploy
+```
+
+If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
```
Comment (fryorcraken) [inline README.md:41]: Maybe worth clarifying that we are using Vercel and providing links to Vercel pipeline + what branch gets deployed where

File: docs/choose-content-topic.md [hunk 2]
```diff
@@ -1,30 +0,0 @@
----
```
Comment (fryorcraken) [inline docs/choose-content-topic.md:1]: Can you keep this file please.

File: docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  tagline: 'Waku is the communication layer for Web3.',
+  url: 'https://waku.guides',
+  baseUrl: '/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: 'waku.org', // Usually your GitHub org/user name.
```
Comment (fryorcraken) [inline docusaurus.config.js:19]:
```suggestion
  organizationName: 'waku-org', // Usually your GitHub org/user name.
```

File: docusaurus.config.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    ({
+      colorMode: {
+        defaultMode: 'dark',
+        disableSwitch: false,
+        respectPrefersColorScheme: true
+      },
+      navbar: {
+        title: 'Waku',
+        logo: {
+          alt: 'My Site Logo',
```
Comment (fryorcraken) [inline docusaurus.config.js:58]: Waku's logo

File: docusaurus.config.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        },
+        items: [
+          {
+            type: 'doc',
+            docId: 'index',
+            position: 'left',
+            label: 'Guides',
+          },
+          {
+            href: 'https://github.com/facebook/docusaurus',
```
Comment (fryorcraken) [inline docusaurus.config.js:69]: to be updated

**Review by fryorcraken — COMMENTED** [review]
I started to review but I guess  it's not there yet. Let me know how I can help.

**Comment by amirhouieh** [comment]
> 

I am not sure why this happens but it seems that Vercel bot keeps trying to deploy mkdocs (already deleted) and that breaks the pipeline. @fryorcraken could you try uninstalling Vercel bot at the settings?https://github.com/waku-org/waku.guide/settings/installations

**Pull request merged**